app/components/works/dates_component.html.erb

```diff
@@ -8,7 +8,8 @@
       <div class="row">
         <div class="col">
           <label for="work_published_year" class="visually-hidden">Publication year</label>
-          <%= number_field_tag 'work[published(1i)]', nil, id: 'work_published_year', class: "form-control", max: Date.today.year %>
+          <%= number_field_tag 'work[published(1i)]', nil, id: 'work_published_year', class: "form-control", min: Settings.earliest_publication_year, max: Time.zone.today.year %>
```

Comment:
This was a super easy validation. Just needed to add a `min:` param to `number_field_tag`. Very cool. Haven't seen this sort of built-in browser/HTML validation before.

Then it was just a matter of include the `invalid-feedback` `div` immediately below with a corresponding error message. Super easy client-side validation without any fuss.